source/includes/steps-install-bi-connector-debian.yaml

```diff
@@ -6,6 +6,72 @@ source:
   ref: download-bi
 ---
 stepnum: 2
+title: Verify the integrity of the downloaded package.
+ref: verify-bi-debian-package
+level: 4
+content: |
+  The MongoDB release team digitally signs all software packages to
```

Comment:
Technically the SQL Engines team is the one signing / releasing these packges, we don't go through the release team to do so, so maybe worth updating this line (here and in the subsequent pages)?